pkg/management/mock/instrumenting.go

```diff
@@ -35,16 +35,19 @@ func (m *Histogram) EXPECT() *HistogramMockRecorder {
 
 // Observe mocks base method
 func (m *Histogram) Observe(arg0 float64) {
+	m.ctrl.T.Helper()
```

Comment:
You shall use the following command as it seems that linux developpers are not supporting m.ctrl.T.Helper()
`find . -path "*/vendor" -prune -o -path "*/mock/*" -name "*.go" -type f -exec sed -i '/Helper()/d' {} \;`

Reply:
Done